TowerForge/TowerForge/Statistics/StatisticsFactory.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  StatisticsFactory.swift
+//  TowerForge
+//
+//  Created by Rubesh on 12/4/24.
+//
+
+import Foundation
+
+class StatisticsFactory {
+    static let availableEventTypes: [TFEvent.Type] = [
```

Comment:
i think this enum is better positioned in the events class?

Reply:
Since TFEvent is a protocol I get the "Static member 'availableEventTypes' cannot be used on protocol metatype '(any TFEvent).Type'" which I'm not too sure how to resolve, so I just moved it to the ObjectSet

Reply:
is moving it to `eventmanager` better?